eth_abi/decoding.py

```diff
@@ -24,6 +27,47 @@
 )
 
 
+class ContextFramesBytesIO(io.BytesIO):
+    """
+    A byte stream which can track a series of contextual frames in a stack.
+    """
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._frames = [(0, 0)]
```

Comment:
I don't understand this initial value.  Looking at the code there doesn't appear to be a code path that can even touch `self._frames[0]`?

Reply:
Yep, you're right.  I need to take that out.  I had it that way because, previously, I was referencing the current context frame by doing `self._frames[-1]`.  But then I began using `self._total_offset` instead and forgot to reconsider the use of a default first frame.